packages/react-bindings/src/hooks/useStyles.ts

```diff
@@ -81,7 +80,7 @@ const useStyles = <StyleProps extends PrimitiveProps>(
     rtl,
     saveDebug: fluentUIDebug => (debug.current = { fluentUIDebug }),
     theme: context.theme,
-    _internal_resolvedComponentVariables: context._internal_resolvedComponentVariables,
+    __experimental_cache: true,
```

Comment:
We can add configuration for whether the caching should be enabled if needed, for now it is always true when using the useStyles hook